scripts/automated_ingestion/eessi_task.py

```diff
@@ -817,10 +817,12 @@ def _create_task_summary(self) -> str:
         # create task summary
         payload_name = self.description.metadata["payload"]["filename"]
         payload_summary = self.payload.analyse_contents(self.config)
+        payload_verified_icon = " :closed_lock_with_key:" if self.payload.signature_verified else ":warning:"
```

Comment:
Maybe also add a whitespace before the `:warning:`
```suggestion
        payload_verified_icon = " :closed_lock_with_key:" if self.payload.signature_verified else " :warning:"
```